src/celengine/boundaries.cpp

```diff
@@ -74,8 +74,8 @@ ReadBoundaries(std::istream& in)
             currentChain.clear();
         }
 
-        currentChain.emplace_back(astro::equatorialToCelestialCart(ra, dec,
-                                                                   BoundariesDrawDistance));
+        currentChain.emplace_back(celestia::astro::equatorialToCelestialCart(ra, dec,
```

Comment:
better consistently use `namespace astro = celestia::astro;`, then one day when all code will be in `celestia` namespace no changes will be required.